packages/muelu/doc/TutorialReadTheDocs/pages/level_smoothers.rst

```diff
@@ -0,0 +1,148 @@
+.. _level_smoothers:
+
+===============
+Level smoothers
+===============
+
+From the last tutorial we have learned that the used multigrid algorithm may have a significant influence in the convergence speed. When comparing the error plots for the standalone multigrid smoothers with unsmoothed and smoothed aggregation multigrid one finds also a notable difference in the **smoothness** of the error.
+
+Background on multigrid methods
+===============================
+
+Obviously there are cases where some highly oscillatory error modes are left and overlaying some low frequency modes. In other cases there are only low frequency error modes left. Theses are two typical cases one might find in practice.
+
+Multigrid methods are based on the fact, that (cheap) level smoothing method often are able to smooth out high oscillatory error components whereas they cannot reduce low frequency error components very well. These low frequency error components then are transferred to a coarse level where they can be seen as high frequency error component for a level smoother on the coarse level.
+
+.. warning:
+
+  Display an image of a sine wave  appearing oscillatory on a coarse grid
```

Comment:
I added this code here, but I just want to point out that the above statement is one of the most important and fundamental concepts to understand with multigrid, and there should probably be a figure to accompany it. Something similar to Briggs 2e Figure 2.6, but not necessarily a direct copy.

![image](https://user-images.githubusercontent.com/12531152/199308970-bb3b4668-aaf8-4b2f-b9b8-5014959f5d5b.png)

Reply:
You're right! We should not copy this, but rather make our own version of such a picture.

@maxfirmbach, do you have such an introductory picture available, that we could reuse here?